extensions/amp-soundcloud/0.1/test/test-amp-soundcloud.js

```diff
@@ -36,83 +36,95 @@ describes.realWin(
       doc = win.document;
     });
 
-    function getIns(mediaid, playlist, opt_attrs) {
-      const ins = doc.createElement('amp-soundcloud');
-      if (playlist) {
-        ins.setAttribute('data-playlistid', mediaid);
+    function getSCPlayer(mediaid, isPlaylist, opt_attrs) {
+      const scplayer = doc.createElement('amp-soundcloud');
+      if (isPlaylist) {
+        scplayer.setAttribute('data-playlistid', mediaid);
       } else {
-        ins.setAttribute('data-trackid', mediaid);
+        scplayer.setAttribute('data-trackid', mediaid);
       }
-      ins.setAttribute('height', '237');
+      scplayer.setAttribute('height', '237');
 
       if (opt_attrs) {
         for (const attr in opt_attrs) {
-          ins.setAttribute(attr, opt_attrs[attr]);
+          scplayer.setAttribute(attr, opt_attrs[attr]);
         }
       }
 
-      doc.body.appendChild(ins);
-      return ins
+      doc.body.appendChild(scplayer);
+      return scplayer
         .build()
-        .then(() => ins.layoutCallback())
-        .then(() => ins);
+        .then(() => scplayer.layoutCallback())
+        .then(() => scplayer);
     }
 
     it('renders track', () => {
-      return getIns('243169232').then(ins => {
-        const iframe = ins.firstChild;
+      return getSCPlayer('243169232').then(scplayer => {
+        const iframe = scplayer.firstChild;
         expect(iframe).to.not.be.null;
         expect(iframe.tagName).to.equal('IFRAME');
         expect(iframe.src).to.equal(trackEmbedUrl);
       });
     });
 
     it('renders playlist', () => {
-      return getIns('173211206', true).then(ins => {
```

Comment:
I noticed this number changed. Was there something wrong with `173211206` vs `1595551`?

Reply:
The playlist doesn't exist anymore, replaced it with one from the official soundcloud playground